src/Resources/app/administration/src/component/adyen-config-check-button/index.js

```diff
@@ -41,12 +41,14 @@ Component.register('adyen-config-check-button', {
 
     computed: {
         pluginConfig() {
+            debugger;
```

Comment:
Can we please remove debugger.

Reply:
Thanks for the heads-up, debugger has been removed.